conviva/src/main/java/com/bitmovin/analytics/conviva/ssai/DefaultSsaiApi.java

```diff
@@ -71,12 +81,16 @@ public void reportAdStarted(AdInfo adInfo) {
             Log.d(TAG, "No server side ad break active");
             return;
         }
+        if (playerAdapter == null) {
+            Log.d(TAG, "Player not yet set. Cannot report ad started.");
```

Comment:
Should probably be a warning.

Reply:
[e709ddd](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/80/commits/e709dddb6e463aa615cd3e14d040205b060530e2)